### DISCUSSION TITLE
Triple H er kominn í regluna - 2023/5/3

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#12] [vefsida-ummæli]
# Triple H er kominn í regluna - 2023/5/3

Hinn eini sanni Triple H er kominn í regluna, þetta er mjög spennandi

https://www.biskupsreglan.com/frettir/halli-kominn-i-hopinn

**Comment by MatthiasGudni** [comment]
Jibbí
